mipengine/node/monetdb_interface/guard.py

```diff
@@ -102,6 +102,7 @@ class InvalidSQLParameter(Exception):
 alpha_re = r"[a-zA-Z]"
 alnum_re = r"[a-zA-Z0-9]"
 alnumhyphen_re = r"[a-zA-Z0-9-]"
+alnumhyphen_ptrn = re.compile(f"{alnumhyphen_re}*", re.A)
```

Comment:
This matches the string `--` which is a comment in SQL. This is typically used in SQL injection attacks. 

If `request_id` is a uuid why not using a uuid regex? 
```
uuid_re = r"0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```
If we need non-uuid as well (I guess to avoid breaking tests) we can use
```
rf"{uuid_re}|\w+"
```
or something like that.